ppr-ui/src/App.vue

```diff
@@ -388,17 +388,18 @@ export default class App extends Mixins(AuthMixin) {
     let statusCode = StatusCodes.OK
     try {
       const authRoles = getKeycloakRoles()
+      let isSbc
       if (authRoles && authRoles.length > 0) {
         if (authRoles.includes('gov_account_user')) {
           // if staff make call to check for sbc
-          const isSbc = await getSbcFromAuth()
+          isSbc = await getSbcFromAuth()
           this.setRoleSbc(isSbc)
         }
         if (!authRoles.includes('ppr') && !authRoles.includes('mhr')) {
           throw new Error('No access to Assets')
         }
         this.setAuthRoles(authRoles)
-        this.setUserAccessRole(authRoles)
+        this.setUserAccessRole({ authRoles: authRoles, isSbc })
```

Comment:
Sort of same question as below, would the state getter that exists already for identifying the SBC role not suffice?
Just curious why we apply it to another array when it's already defined in store as `stateModel.authorization.isSbc`

Reply:
It would suffice, I thought calling a `isRoleStaffSbc` in `setUserAccessRole` setter didn't look clean to me, so I thought just to pass it down as a prop.

Reply:
For sure, I definitely agree with that aspect but same as above, not referring to its use of a prop here, more a question do we need to add it to this array at all, since there already is a global getter which will identify this role.


It seems like we have `UserAccessRole` driving some conditionals in the UI and those other global getters driving conditions elsewhere.

Is the end goal to have this array drive all our role based conditionals?

Reply:
Cleaned up.